src/ert/_c_wrappers/enkf/queue_config.py

```diff
@@ -5,8 +5,8 @@
 from dataclasses import dataclass, field
 from typing import Dict, List, Tuple, Union
 
-from ert._c_wrappers.config.config_parser import ConfigValidationError
-from ert._c_wrappers.job_queue import Driver, JobQueue, QueueDriverEnum
+from ..config import ConfigValidationError
```

Comment:
Why are these relative now? I have no idea what is best, but it seems we have absolute in most places

Reply:
Restored to old state as it is not really that related to the commit